health-sdk/sdk/src/doc/source/flow.rst

```diff
@@ -164,6 +164,7 @@ The ``PaymentFlowConfiguration`` class contains the following options:
 - ``shouldShowReviewBottomDialog``: If set to ``true``, the ``PaymentFragment`` will show a bottom sheet dialog containing the payment details. If set to ``false``, the payment details will not be visible during the payment flow. They will be available to be reviewed after redirecting to the selected payment provider,
     before finalizing the payment. This parameter is only used in the case of payment flows started without ``documentId``.
     Default value is ``false``
+- ``popupDurationPaymentReview``: Defines the duration (in seconds) for which the payment review popup will be displayed before automatically closing. This value should be within a reasonable range (e.g., 0-10 seconds) to ensure a smooth user experience. Default value is ``3`` seconds.
```

Comment:
[Request] Because we coerce the value to be between 0 and 10, please specify this as a must, rather than a suggestion.